README.md

```diff
@@ -18,8 +20,10 @@ These templates are not an exhaustive list of use cases or possibilities that de
 There are two ways you can use the templates provided in this repository. 
 
 1. Using SnowCLI (recommended)
+
+    Please refer to the online Snowflake Documentation on installing and using SnowCLI to create Native Applications for more information. 
     
-    Run the following command in your terminal.
+    Assuming you have everything set up, run the following command in your terminal.
```

Comment:
Feels a bit vague. Maybe:
Assuming [...] set up, -> Once SnowCLI is installed and configured